docs/content/en/docs/yaml-crd-ref/evaluationdefinition.md

```diff
@@ -59,11 +59,22 @@ spec:
 ## Usage
 
 A `KeptnEvaluationDefinition` references one or more
-[KeptnMetric](metric.md) CRDs.
+[KeptnMetric](metric.md) resources.
 When multiple `KeptnMetric`s are used,
 the Keptn Lifecycle Toolkit considers the evaluation successful
 if **all** metrics meet their `evaluationTarget`.
 
+The `KeptnMetric` resource and associated
+[KeptnMetricsProvider](metricsprovider.md)
+resource must be located in the same namespace
+but the `KeptnEvaluationDefinition` resources
+can reference metrics from any namespace in the cluster.
+This means that you can create `KeptnMetricsProvider`
+and `KeptnMetric` resources
+in a cluster-wide namespace (e.g. in `keptn-lifecycle-toolkit-system`)
```

Comment:
```suggestion
in a central namespace (e.g. in `keptn-lifecycle-toolkit-system`)
```
"cluster-wide" could be misleading here. there are actual cluster-wide resources that don't belong to a namespace, so I wouldn't use that term here, since it has a commonly accepted meaning that is different from what we are trying to say here

Reply:
Interesting.  I modified the comment to use "centralized" rather than "central" -- it's better English and I hope is doesn't screw up the technical accuracy.